cache/api/src/main/java/org/apache/karaf/cache/api/CacheService.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.karaf.cache.api;
```

Comment:
Missing ASF header here.

Reply:
It's missing in other places too (still a draft), will fix everything once we know how the interface should look like.